crates/context/src/lib.rs

```diff
@@ -122,6 +129,7 @@ impl ContextManager {
     pub async fn join_context(
         &self,
         context_id: &calimero_primitives::context::ContextId,
+        initial_identity: KeyPair,
```

Comment:
I truly do not understand why fully-qualified names are used everywhere, but I was told it was to avoid ambiguity 😅 

Regardless of my feelings about the practice, there is no ambiguity here as there is only one type with that name...?